core/matching/orderbook.go

```diff
@@ -126,6 +126,11 @@ func (b *OrderBook) ReloadConf(cfg Config) {
 	b.cfgMu.Unlock()
 }
 
+// GetOrderBookLevelCount returns the number of levels in the book.
+func (b *OrderBook) GetOrderBookLevelCount() uint64 {
+	return uint64(len(b.ordersByID))
```

Comment:
Sanity check, this looks like returning the number of orders, not the number of levels?

Reply:
For me that's not a killer by the way - if number of orders is much easier - I would be happy to change the spec draft. 

I don't know what causes the cost - whether it's more the number of levels or more the number of orders.